data/__init__.py

```diff
@@ -151,377 +151,403 @@
+                                      u"\u7f1d\u5408\u602a /// Stitches", u"\u7e2b\u5408\u602a /// Stitches"],
+                         "Sylvanas": [u"Sylvanas", u"Sylvanas /// Sylvanas", u"Sylvana /// Sylvanas",
+                                      u"Sylvanas /// Sylvanas", u"Sylvanas /// Sylvanas", u"Sylvanas", u"Sylvanas",
+                                      u"\uc2e4\ubc14\ub098\uc2a4 /// Sylvanas", u"Sylwana /// Sylvanas",
+                                      u"\u0421\u0438\u043b\u044c\u0432\u0430\u043d\u0430 /// Sylvanas",
+                                      u"\u5e0c\u5c14\u74e6\u5a1c\u65af /// Sylvanas",
+                                      u"\u5e0c\u74e6\u5a1c\u65af /// Sylvanas"],
+                         "Tassadar": [u"Tassadar", u"Tassadar /// Tassadar", u"Tassadar /// Tassadar",
+                                      u"Tassadar /// Tassadar", u"Tassadar /// Tassadar", u"Tassadar /// Tassadar",
+                                      u"Tassadar /// Tassadar", u"\ud0dc\uc0ac\ub2e4\ub974 /// Tassadar",
+                                      u"Tassadar /// Tassadar",
+                                      u"\u0422\u0430\u0441\u0441\u0430\u0434\u0430\u0440 /// Tassadar",
+                                      u"\u5854\u8428\u8fbe\u5c14 /// Tassadar", u"\u5854\u85a9\u9054 /// Tassadar"],
+                         "Thrall": [u"Thrall", u"Thrall /// Thrall", u"Thrall /// Thrall", u"Thrall /// Thrall",
+                                    u"Thrall /// Thrall", u"Thrall /// Thrall", u"Thrall",
+                                    u"\uc2a4\ub784 /// Thrall", u"Thrall /// Thrall",
+                                    u"\u0422\u0440\u0430\u043b\u043b /// Thrall", u"\u8428\u5c14 /// Thrall",
+                                    u"\u7d22\u723e /// Thrall"],
+                         "Gazlowe": [u"Tinker", u"Gazlowe", u"Gasganete /// Gazlowe", u"Gazlowe /// Gazlowe",
+                                    u"Gazlowe /// Gazlowe", u"Gazleu /// Gazlowe", u"Sparachiodi /// Gazlowe",
+                                    u"\uac00\uc988\ub85c /// Gazlowe", u"Gazol /// Gazlowe",
+                                    u"\u0413\u0430\u0437\u043b\u043e\u0443", u"\u52a0\u5179\u9c81\u7ef4 /// Gazlowe",
+                                    u"\u52a0\u8332\u9b6f\u7dad /// Gazlowe"],
+                         "Tracer": [u"Tracer", u"Tracer /// Tracer", u"Tracer /// Tracer", u"Tracer",
+                                    u"Tracer /// Tracer", u"Tracer", u"Tracer",
+                                    u"\ud2b8\ub808\uc774\uc11c /// Tracer", u"Smuga",
+                                    u"\u0422\u0440\u0435\u0439\u0441\u0435\u0440", u"\u730e\u7a7a",
+                                    u"\u9583\u5149 /// Tracer"],
+                         "Tychus": [u"Tychus", u"Tychus", u"Tychus /// Tychus", u"Tychus /// Tychus",
+                                    u
```

Comment:
so I'm not recreating to inverse mapping each time a new Character class is loaded.